src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -721,9 +721,23 @@
      * @see FullIdent
      **/
     public static final int METHOD_CALL = GeneratedJavaTokenTypes.METHOD_CALL;
-
```

Comment:
Done. BTW I noticed that there are breaks after some definition while no breaks after the others in this file. Is there a specific rule of whether to put a break after a definition?